lib/native/aws_iot.ts

```
@@ -70,34 +71,69 @@ export class AwsIotMqttConnectionConfigBuilder {
         return builder;
     }
 
-    with_certificate_authority_from_path(ca_path?: string, ca_file?: string) {
+    /**
+     * Overrides the default system trust store.
+     * @param ca_dirpath - Only used on Unix-style systems where all trust anchors are
+     * stored in a directory (e.g. /etc/ssl/certs).
+     * @param ca_filepath - Single file containing all trust CAs, in PEM format
+     */
+    with_certificate_authority_from_path(ca_dirpath?: string, ca_filepath?: string) {
         if (this.tls_ctx_options !== undefined) {
-            this.tls_ctx_options.override_default_trust_store(ca_path, ca_file);
+            this.tls_ctx_options.override_default_trust_store_from_path(ca_dirpath, ca_filepath);
         }
```

likewise, weird that this silently fails to have any effect